ngi_reports/stockholm/project_summary.py

```diff
@@ -114,22 +114,24 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 self.LOG.info('Sample {} is aborted, so skipping it'.format(sample_id))
                 self.project_info['aborted_samples'][sample_id] = {'user_id': sample.get('customer_name',''), 'status':'Aborted'}
                 continue
-            
+
             ## Basic fields from Project database
             self.samples_info[sample_id] = {'ngi_id': sample_id}
-            ## special characters should be removed 
-            self.samples_info[sample_id]['customer_name'] = self.to_ascii(sample.get('customer_name',''))
-            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)')
-            self.samples_info[sample_id]['reads_min'] = sample.get('details',{}).get('reads_min')
-            self.samples_info[sample_id]['preps'] = {}
-            
-            ## Check if a non-aborted sample is sequenced
-            if self.samples_info[sample_id]['total_reads'] == None:
+
+            ## get total reads if avialable or mark sample as not sequenced
+            try:
+                self.samples_info[sample_id]['total_reads'] = str(round(float(sample.get('details',{})['total_reads_(m)']), 2))
```

Comment:
instead of `str(round(float(` maybe just use string formatting?

```python
'{:.2f}'.format(sample.get('details',{})['total_reads_(m)']))
```

Also, I think that this will raise a `KeyError` if the value isn't there - the default value is an empty dict (`{}`), which won't have the `total_reads_(m)` key.. You catch this anyway, which means that you probably don't the `get`:

```python
'{:.2f}'.format(sample['details']['total_reads_(m)'])
```

Much cleaner! :grinning: